DESCRIPTION

```diff
@@ -40,8 +41,11 @@ Imports:
 Suggests: 
     covr (>= 3.2.1),
     english (>= 1.2.3),
+    here(>= 0.1),
+    kableExtra (>= 1.1.0),
     lintr (>= 1.0.3),
     testthat (>= 2.1.0),
+    tidyverse (>= 1.2.1),
```

Comment:
tidyverse best practices are to not use the actual **tidyverse** package in the DESCRIPTION field. Instead, it's better to list the individual packages you need. See [here](https://www.tidyverse.org/articles/2018/06/tidyverse-not-for-packages/).